### ISSUE TITLE
Screen reader is not announcing the 'selected/unselected' state information for 'Favorite food' combo box in Windows.

### ISSUE THREAD

**Issue by msftedad** [angular/components#30034]
### Is this a regression?

- [x] Yes, this behavior used to work in the previous version

### Description

Title: Screen reader is not announcing the 'selected/unselected' state information for 'Favorite food' combo box in Windows.

Observation:
Here, all three Screen readers are announcing incorrect information as selected for the list items in the combo box.

User Impact:
Screen Reader users will find difficulty in getting the required information if Screen Reader is not announcing selected state for the selected list item.

### Reproduction

Pre-Requisite:
Turn on Screen Readers:
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J
Narrator: Ctrl+ Alt + Enter
Verbosity:
JAWS: Beginner, Highest
NVDA: Default
Narrator: Default

Repro steps: 

1. Open URL: https://v9.material.angular.io/components/select/overview in latest chrome browser.
2. Press tab key to reach 'Favorite food' combo box under 'Basic select' and expand it using enter key.
3. Verify with down arrow key whether screen reader is announcing selected/unselected state for 'Favorite food' combo box or not.

### Expected Behavior

Screen reader should announce the 'selected/unselected' state information for 'Favorite food' combo box. It should announce it as 'Steak 1 of 3 selected'.

### Actual Behavior

While navigating using down arrow key, Screen reader is not announcing the 'selected/unselected' state information for 'Favorite food' combo box. It is announcing as 'Steak 1 of 3'.

![Image](https://github.com/user-attachments/assets/c279b2c8-d6c0-425e-8564-f41e5ebb656e)
https://github.com/user-attachments/assets/f2f7d3fa-477d-404a-9025-3aed2f96838c

### Environment

- Angular:  9.2.4
- CDK/Material:9.2.4
- OS: Windows 11 Version 24H2(OS Build 26120.1542)
- Browser: Chrome: Version 129.0.6668.90 (Official Build) (64-bit)

**Comment by naaajii** [comment]
seems like duplicate of #29040.

**Comment by crisbeto** [comment]
Closing as a duplicate.
